src/System.Text.Encodings.Web/src/System/Text/Unicode/UnicodeRanges.cs

```diff
@@ -17,31 +17,31 @@ public static partial class UnicodeRanges
         /// An empty <see cref="UnicodeRange"/>. This range contains no code points.
         /// </summary>
         public static UnicodeRange None => _none ?? CreateEmptyRange(ref _none);
-        private static UnicodeRange _none;
```

Comment:
Probably not in scope for the current change, but statics do not follow naming convention here. Worth renaming?

Reply:
Not as part of this change, but separately would be fine.

Reply:
The backing file is generated by a tool. If you want to regenerate this file, update the tool sources at https://github.com/dotnet/corefx/blob/master/src/System.Text.Encodings.Web/tools/GenUnicodeRanges/Program.cs and rerun it.

Reply:
Thanks, I would have completely missed that.